coalib/parsing/LineParser.py

```diff
@@ -82,6 +82,15 @@ def _parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
+        if '#' in comment:
+            comment_index = comment.index('#')
+            if comment_index + 1 < len(comment) or comment_index - 1 > 0:
+                if comment[comment.index('#') + 1] != ' ' \
```

Comment:
If your if statement is spanning across multiple lines you may use braces with if `if( )`